Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonCustomPaletteBase.cs

```diff
@@ -2121,6 +2121,7 @@ public void PopulateFromBase(bool silent)
         public string? Import(string filename, bool silent)
         {
             string? ret;
+            string outputPath = $@"{Application.ExecutablePath}\\{filename}_Upgraded.xml";
```

Comment:
Has to be in memory (Or at worst, a proper temporary file that is auto deleted after use)

Reply:
Reverted,  needs some thinking through, maybe `StreamWriter`?